src/witan/app/forecast.clj

```diff
@@ -116,13 +126,70 @@
                             :owner owner
                             :in_progress false}))
 
+
+
+(defn add-to-result-values
+  [result name value]
+  (assoc result :values (assoc (:values result) name (hayt/user-type {:name name :value value}))))
```

Comment:
actually, there's `update` in clojure 1.7.0:

`(update result :values assoc name (hayt/user-type {:name name :value value}))`